### DISCUSSION TITLE
Opens a new untitled file after closing without saving.

### DISCUSSION THREAD

**Discussion by reinew** [community/community#39094] [Copilot Conversations]
### Select Topic Area

Bug

### Body

If i create a new untitled file, choose a language for example javascript, type in some code and then close the file and choose "Don't save" in the popup, a couple of seconds later a new untitled file opens up by itself.

When i disable the Copilot extension, this issue don't happen.
I used Bisesct to find out that is was the Copilot extension that causes the problem.

I tried to file an issue report from within vscode but nothing happened when i clicked the "Create on GitHub" button...?

**Comment by iamcjwilliams** [comment]
@reinew Do you have the latest version of the extension?

**Reply by reinew** [reply]
Yes, just updated to v1.59.7338
Last released 2022-11-15 22:39:57
Last updated 2022-11-16 01:07:28

This is a new version from what i had when i first posted my problem two days ago but the problem remains.

My IDE is vscode v1.73.1 on a macbook air M1.

**Comment by iamcjwilliams** [comment]
@reinew Our copilot team will review this issue

**Reply by Mariownyou** [reply]
Any updates?

**Comment by svipas** [comment]
Sounds like related to https://github.com/orgs/community/discussions/13901. And, this issue is happening from the beginning as I remember and still till this day, very annoying that it creates random Untitled files. Why it does that?

**Comment by Mariownyou** [comment]
still a problem

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/73722

**Comment by Ste1io** [comment]
Why is this closed? this is a MAJOR PROBLEM on latest (un)stable build. And easy af to repro.

- Open a workspace (it must be a workspace)
- Now open a new untitled text window (`CTRL`+`N`)
- Now paste some text (JSON works) to trigger a language mode change (or set the language mode yourself)
- Now close the untitled window, selecting "No" in the save prompt

Even better, open `C:\Users\vibecoder\AppData\Roaming\Code\Backups\{workspaceIdentifier.id}\untited` and watch the fun happen in windows exploder while you play wack-a-tab.

(Direct link to that directory is in `C:\Users\vibecoder\AppData\Roaming\Code\User\globalStorage\storage.json` listed for each open vscode instance).
